src/pages/posts/[slug].page.tsx

```diff
@@ -36,10 +38,11 @@ const PostDetails = ({ post }: PostProps) => {
 
       <h1
         className="m-8 text-7xl items-center text-center"
-        data-test-id={postDetailsTestIds.title}
+        data-test-id={postDetailsTestIds.homePageLink}
       >
-        {post.title}
+        <Link href="/">TIL</Link>
```

Comment:
Should h1 be the title of the post instead? 
And the home page link could be h2 🤔

Reply:
As we are using `text-*` classes, I resized them in 8079f722b4ac21f0a058ce1a981817c8191b3f98, Thank you!

Reply:
Should the `TIL` text on the Post Details page be the same as `TIL` on the home page?